SharedPackages/VPN/Tests/VPNTests/NetworkProtectionDeviceManagerTests.swift

```diff
@@ -109,27 +109,6 @@ final class NetworkProtectionDeviceManagerTests: XCTestCase {
         XCTAssertEqual(networkClient.spyRegister?.requestBody.server, server.serverName)
     }
 
-    func testWhenGeneratingTunnelConfig_storedAuthTokenIsInvalidOnGettingServers_deletesToken() async {
-        _ = NetworkProtectionServer.mockRegisteredServer
-        networkClient.stubRegister = .failure(.invalidAuthToken)
-
-        XCTAssertNotNil(tokenHandler.token)
-
-        _ = try? await manager.generateTunnelConfiguration(selectionMethod: .automatic, regenerateKey: false)
-
-        XCTAssertNil(tokenHandler.token)
-    }
-
-    func testWhenGeneratingTunnelConfig_storedAuthTokenIsInvalidOnRegisteringServer_deletesToken() async {
-        networkClient.stubRegister = .failure(.invalidAuthToken)
-
-        XCTAssertNotNil(tokenHandler.token)
-
-        _ = try? await manager.generateTunnelConfiguration(selectionMethod: .automatic, regenerateKey: false)
-
-        XCTAssertNil(tokenHandler.token)
-    }
-
```

Comment:
These were failing for me locally, and I don't think that their behavior is accurate any more - pretty sure we just throw an error if this case ever happens?